### PR TITLE
Add `--replica` flag to create password

### PR DESCRIPTION
**Pull request by notfelineit**
This PR adds `--replica` flag to create password, which when enabled, will create a password that redirects queries to the replica tablets, or the  branch's read-only-regions, depending on where latency is smallest.

### CODE REVIEW

File: internal/cmd/password/create.go [hunk 1]
```diff
@@ -76,6 +78,8 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 	cmd.PersistentFlags().StringVar(&flags.role, "role",
 		"admin", "Role defines the access level, allowed values are : reader, writer, readwriter, admin. By default it is admin.")
 	cmd.PersistentFlags().Var(&flags.ttl, "ttl", `TTL defines the time to live for the password. Durations such as "30m", "24h", or bare integers such as "3600" (seconds) are accepted. The default TTL is 0s, which means the password will never expire.`)
+	cmd.Flags().BoolVar(&flags.replica, "replica", false, "When enabled, the password will route all reads to the branch's primary replicas, and all read-only regions.")
+	cmd.Flags().MarkHidden("replica")
```
Comment (notfelineit) [inline internal/cmd/password/create.go:82]: We may be able to remove this.

File: internal/cmd/password/create.go [hunk 2]
```diff
@@ -76,6 +78,8 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 	cmd.PersistentFlags().StringVar(&flags.role, "role",
 		"admin", "Role defines the access level, allowed values are : reader, writer, readwriter, admin. By default it is admin.")
 	cmd.PersistentFlags().Var(&flags.ttl, "ttl", `TTL defines the time to live for the password. Durations such as "30m", "24h", or bare integers such as "3600" (seconds) are accepted. The default TTL is 0s, which means the password will never expire.`)
+	cmd.Flags().BoolVar(&flags.replica, "replica", false, "When enabled, the password will route all reads to the branch's primary replicas, and all read-only regions.")
```
Comment (iheanyi) [inline internal/cmd/password/create.go:81]:
```suggestion
	cmd.Flags().BoolVar(&flags.replica, "replica", false, "When enabled, the password will route all reads to the branch's primary replicas and all read-only regions.")
```

File: internal/cmd/password/password.go [hunk 3]
```diff
@@ -107,6 +110,7 @@ func toPassword(password *ps.DatabaseBranchPassword) *Password {
 		Username:  password.Username,
 		Role:      password.Role,
 		RoleDesc:  toRoleDesc(password.Role),
+		Replica:   password.Replica,
```
Comment (iheanyi) [inline internal/cmd/password/password.go:113]: I wonder if we should do something like `Connection type: Primary / Replica` here instead of a boolean?

**Review by iheanyi — APPROVED** [review]

File: internal/cmd/password/password.go (hunk 3 above)
Comment (notfelineit) [reply internal/cmd/password/password.go:113]: Ooh, yeah!

**Review by iheanyi — APPROVED** [review]

**Pull request merged**